docs/api-reference/Admin.md

```diff
@@ -17,6 +17,9 @@
 ## TabulatorTable(name: str, config: str) -> None  {#TabulatorTable}
 
 
+## Bucket(name: str, title: str, icon\_url: Optional[str], description: Optional[str], overview\_url: Optional[str], tags: Optional[List[str]], relevance\_score: int, last\_indexed: Optional[datetime.datetime], sns\_notification\_arn: Optional[str], scanner\_parallel\_shards\_depth: Optional[int], skip\_meta\_data\_indexing: Optional[bool], file\_extensions\_to\_index: Optional[List[str]], index\_content\_bytes: Optional[int], prefixes: List[str]) -> None  {#Bucket}
+
+
```

Comment:
@sir-sigurd how to make codegen add the buckets module here?

Reply:
https://github.com/quiltdata/quilt/blob/1879f1eb66212dbaa6af8817a50119f54be15fcd/gendocs/pydocmd.yml#L34-L39